### ISSUE TITLE
normalize() does not respect inputRange argument

### ISSUE THREAD

**Issue by ornelles** [aoles/EBImage#25]
The result from `normalize()` does not seem to behave as describe for values handed to the `inputRange` arguments  Here is an example (after loading EBImage_4.18.2):
```R
> x <- matrix(1:9, nrow = 3)
> x <- x/1e3
> x[5] <- 0.1
> x <- Image(x)
imageData(x) # simple "image"
     [,1]  [,2]  [,3]
[1,] 0.001 0.004 0.007
[2,] 0.002 0.100 0.008
[3,] 0.003 0.006 0.009
> round(imageData(normalize(x)), 3) # Expected results
     [,1]  [,2]  [,3]
[1,] 0.00 0.030 0.061
[2,] 0.01 1.000 0.071
[3,] 0.02 0.051 0.081
> xn <- normalize(x, inputRange = c(0, 0.011))
> round(imageData(xn), 3)
     [,1]  [,2]  [,3]
[1,] 0.01 0.040 0.071
[2,] 0.02 1.010 0.081 <-- ???
[3,] 0.03 0.061 0.091
```
Not only was the value > 0.011 **not** clipped but it was replaced with a value **greater** than 1. More extreme values were generated with actual data including images with values ranging from -2 to +5!

(Perhaps not the right place, but thank you very much for creating and maintaining an outstanding set of tools!)

**Comment by aoles** [comment]
Thanks for reporting this, indeed looks like a bug to me. I will fix this soon.

**Comment by ornelles** [comment]
Thanks, again. Best wishes with the fix.

David Ornelles
Sent from iPhone

On Oct 5, 2017, at 7:54 AM, Andrzej K. Oleś <notifications@github.com<mailto:notifications@github.com>> wrote:


Closed #25<https://github.com/aoles/EBImage/issues/25> via fa59e2c<https://github.com/aoles/EBImage/commit/fa59e2c5b18a9e6f556d925160ddb6fe0f7ce4ca>.
